### ISSUE TITLE
[camelcase] definition for rule '@typescript-eslint/camelcase' was not found

### ISSUE THREAD

**Issue by lovekaizen** [davidjbradshaw/eslint-config-adjunct#43]
`eslint-config-airbnb-typescript` has not yet been updated to the latest version of `eslint-typescript` 

A workaround is to add these rules to your eslint config file:

```
"@typescript-eslint/camelcase": "off"
"@typescript-eslint/naming-convention": "warn" // or whichever level you need
```

You can track the PR here: https://github.com/iamturns/eslint-config-airbnb-typescript/issues/95

**Comment by davidjbradshaw** [comment]
What’s wrong with deprecating a rule for one version, before taking out in the following release 🥺

**Comment by lovekaizen** [comment]
this has now been resolved here:  in version `8.0.0` https://github.com/iamturns/eslint-config-airbnb-typescript/pull/99
